### PR TITLE
Issue 25 home screen

### PR DESCRIPTION
**Pull request by piotrmajcher94**

### CODE REVIEW

**Comment by olafkrawczyk** [comment]
Let's use BEM for naming as it's kind of standard right now ;D

File: poker-app-client/public/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,900" rel="stylesheet">
```
Comment (olafkrawczyk) [inline poker-app-client/public/index.html]: Let's create assets folder and put fonts there. Importing them with less will be easy

File: poker-app-client/src/components/home/Home.less [hunk 2]
```diff
@@ -1,44 +1,80 @@
...
+
+      .column-left-content-header {
+        align-items: center;
+        justify-content: center;
+
+        .column-left-content-header__icon {
+          -webkit-animation:spin 4s linear infinite;
+          -moz-animation:spin 4s linear infinite;
+          animation:spin 4s linear infinite;
+          @-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
```
Comment (olafkrawczyk) [inline poker-app-client/src/components/home/Home.less]: We could use autoprefixer to put @-moz and @-webkit for us ;D
https://github.com/less/less-plugin-autoprefix

File: poker-app-client/src/components/home/Home.less [hunk 3]
```diff
@@ -1,44 +1,80 @@
...
+  .home__background {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: @charleston-green;
+    min-width: 100%;
+    min-height: 100%;
+
+    /* polygon */
+    -webkit-clip-path: polygon(100% 0, 100% 21%, 50% 100%, 0 100%, 0 0);
```
Comment (olafkrawczyk) [inline poker-app-client/src/components/home/Home.less]: As above

File: poker-app-client/src/index.less [hunk 4]
```diff
@@ -3,16 +3,105 @@
...
+    font-size: @font-size-regular;
+    font-weight: @font-weight-regular;
+  }
+
+  p.primary-text-large {
+    font-size: @font-size-large;
+    font-weight: @font-weight-light;
+  }
+
+  h1 {
```
Comment (olafkrawczyk) [inline poker-app-client/src/index.less]: This file will grow up pretty fast. I think that we should consider splitting to smaller files. We could move h tags to file typography.less and button to buttons.less and so on

File: poker-app-client/src/components/home/Home.less (hunk 2 above)
Comment (piotrmajcher94) [reply poker-app-client/src/components/home/Home.less]: Looks like create-react-app has already some configuration for Autoprefixer. Let's make another story to figure it out and setup everything properly.

File: poker-app-client/public/index.html (hunk 1 above)
Comment (piotrmajcher94) [reply poker-app-client/public/index.html]: done

File: poker-app-client/src/index.less (hunk 4 above)
Comment (piotrmajcher94) [reply poker-app-client/src/index.less]: done

**Pull request merged**